spec/integration/synced_memory_store/synced_memory_store_spec.rb

```diff
@@ -13,14 +13,15 @@
   # In the test, we wait for the subscriber to make the test more predictable
   let(:logger) { ActiveSupport::TaggedLogging.new(Logger.new(STDOUT)) }
   let!(:subscriber) { SyncedMemoryStore::Subscriber.instance().tap {|i| i.configure(logger: logger).start(wait: true)} }
-  let!(:redis) { Redis.new(url: ENV['REDIS_URL']).tap { |r| r.flushdb }.client }
+  let!(:redis) { Redis.new.tap { |r| r.flushdb }.client }
```

Comment:
We're never setting `REDIS_URL` in specs so by not configuring it we just rely on the default URL